README.md

```diff
@@ -96,17 +90,31 @@ In your web browser, open the link [http://localhost:3000/](http://localhost:300
 # Running applications with Docker
 1. Install Docker if you haven't already
 2. Open Docker Desktop, you cannot run containers or build images, if docker engine is not running
-2. Navigate project root
+2. Now in terminal, navigate to project root
+3. Run `docker compose up --build`, for the first build it may take a while, after build the application will be ran, you can access the application through the `localhost:3000`.
+5. You can stop containers with ctrl+c or using the Docker app
+
+Common Docker commands:
+- If you want to start the application again, run `docker compose up`
+- If you want to just build images, run `docker compose build`
+- If you want to build and run containers, run `docker compose up --build`
+- If you want to build only one service, run `docker compose build <service-name>`, for example `docker compose build backend`
 
 Note: you don't have to initialize submodule to run docker, dockerfile will set up the submodules for you while building the image.
 
-3. Run `docker compose up --build`, for the first build it may take a while, after build the application will be ran, you can access the application through the `localhost:3000`.
-5. You can stop contianers with ctrl+c or using the Docker app
-6. If you want to start the application again, run `docker compose up`
-7. If you want to just build images, run `docker compose build`
-8. If you want to build and run containers, run `docker compose up --build`
 
-Note: If you want to build only one service, use `docker compose build <service-name>`, for example for backend it will be `docker compose build backend`.
+## Working with database
+
+Sometimes you want to run the application without containers, allowing you to develop things quickly. The **Running the iNatAtor Application** section explains how to run the application locally.
+
+You can verify connections are working by going to `localhost:8000/health` and `localhost:8000/echo_env`. Note that these paths are included right now to assist with rapid development, will be removed for production.
```

Comment:
I still would like to keep /health/ because some people don't know how to communicate with postgres through terminal or don't know how to check if things are running correctly on docker, /health/ is an easy way to make sure thing is up and running. echo_env can be removed